selector: method (#8854)
fishtown-analytics/docs.getdbt.com: website/docs/reference/node-selection/yaml-selectors.md

```
</File>

<VersionBlock firstVersion="1.12">

> Starting in dbt Core v1.12, you can reference a named selector from `--select` or `--exclude` using the [`selector` method](/reference/node-selection/methods#selector) (for example, `selector:my_selector`).

Reviewer: ```suggestion
Starting in <Constant name="core" /> v1.12, you can reference a named selector from `--select` or `--exclude` using the [`selector` method](/reference/node-selection/methods#selector) (for example, `selector:my_selector`).
```

The `selector` method returns the complete set of nodes returned by the named selector.

<VersionBlock firstVersion="1.12">

> If selector definitions reference each other in a cycle, dbt raises the `DbtRecursionError` error at runtime. For example, the following selector inheritance is invalid:

Reviewer: ```suggestion
If selector definitions reference each other in a cycle, dbt raises `DbtRecursionError` at runtime. For example, the following selector inheritance is invalid:
```

> ```yml

Reviewer: ```suggestion
```yaml
```

Notes:
- You can combine `--select` with `--exclude` for ad-hoc selection of nodes.
- The `--select` and `--selector` syntax both provide the same overall functions for node selection. Using [graph operators](/reference/node-selection/graph-operators) (such as `+`, `@`.) and [set operators](/reference/node-selection/set-operators) (such as `union` and `intersection`) in `--select` is the same as YAML-based configs in `--selector`.

> - If you use `--selector` together with `--select` or `--exclude`, dbt only applies `--selector` for node selection and ignores `--select` and `--exclude`. Starting in <Constant name="core" /> v1.12, dbt raises the `SelectExcludeIgnoredWithSelectorWarning` when `--selector` is combined with `--select` or `--exclude`.

Author: ```suggestion
- If you use `--selector` together with `--select` or `--exclude`, dbt only applies `--selector` for node selection and ignores `--select` and `--exclude`. Starting in <Constant name="core" /> v1.12, dbt raises the `SelectExcludeIgnoredWithSelectorWarning` when `--selector` is combined with `--select` or `--exclude`. If you want to combine a selector with these flags, use the [`selector:` method](/reference/node-selection/methods#selector) instead.
```

> - If you use `--selector` together with `--select` or `--exclude`, dbt only applies `--selector` for node selection and ignores `--select` and `--exclude`. Starting in <Constant name="core" /> v1.12, dbt raises the `SelectExcludeIgnoredWithSelectorWarning` warning when `--selector` is combined with `--select` or `--exclude`.

Reviewer: ```suggestion
- If you use `--selector` together with `--select` or `--exclude`, dbt only applies `--selector` for node selection and ignores `--select` and `--exclude`. Starting in <Constant name="core" /> v1.12, dbt raises the `SelectExcludeIgnoredWithSelectorWarning` when `--selector` is combined with `--select` or `--exclude`.
```
